notebooks/roman_cutouts/roman_cutouts.ipynb

```diff
@@ -0,0 +1,806 @@
+    "padding = 500\n",
+    "plt.xlim(pix_x - padding, pix_x + padding)\n",
+    "plt.ylim(pix_y - padding, pix_y + padding)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "784020a0-32ff-4d0b-b260-d8e116319df0",
+   "metadata": {},
+   "source": [
+    "The ``asdf_cut`` function takes as input the following: \n",
+    "- `input_file`: The input ASDF filename or URI\n",
+    "- `ra`: The right ascension of the central cutout\n",
+    "- `dec`: The declination of the central cutout\n",
+    "- `cutout_size`: The image cutout pixel size (optional, default `200`)\n",
+    "- `output_file`: The name of the output cutout file. The output format is determined by the filename extension (optional, default `\"example_roman_cutout.fits\"`)\n",
+    "- `write_file`: Flag to write the cutout to a file or not (optional, default `True`)\n",
+    "- `fill_value`: The fill value for any pixels outside the original image (optional, default `np.nan`)\n",
+    "  \n",
+    "Let's create a square cutout, at the center of the image, 200 pixels on a side, and save the output to a new FITS file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b71c7031-ffe6-4941-8a12-25d7b1948937",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# create the image cutout as a FITS file\n",
+    "cutout = asdf_cut(asdf_file_uri, coord.ra, coord.dec, cutout_size=cutout_size, output_file=\"roman-demo.fits\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e1460b04-2620-44f4-97c3-a0e246687e76",
+   "metadata": {},
+   "source": [
+    "### Inspect FITS Cutout Image\n",
+    "Now let's inspect the cutout a bit and display the image cutout with matplotlib.  The ``asdf_cut`` function returns an [Astropy Cutout2D](https://docs.astropy.org/en/stable/nddata/utils.html#overview) object.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "84597c63-0d92-423c-a13a-521bb6249ac1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('shape:', cutout.shape)\n",
+    "print('orginal position:', cutout.position_original)\n",
+    "print('cutout position:', cutout.position_cutout)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a29c636f-4b21-497d-baa2-415818750a27",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with fits.open(\"roman-demo.fits\") as hdulist:\n",
+    "    hdulist.info()\n",
+    "    header = hdulist[0].header\n",
+    "    img = hdulist[0].data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab0ce5d0-df21-4e26-b2ae-d40e7443293b",
+   "metadata": {},
+   "source": [
+    "#### Cutout in Pixel Coordinates"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "69ac82b9-57ab-4b7c-854d-a5e74698d1ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_cutout_pixel(img):\n",
+    "    plt.figure(figsize=(6, 6))\n",
```

Comment:
can you add a single-line docstring here with a small text of what this helper function does?